ui/client/components/graph/EspNode/esp.ts

```diff
@@ -1,134 +1,213 @@
 /* eslint-disable i18next/no-literal-string */
```

Comment:
I think esp is not good enough name.. What kind of elements we keep there? Configs / styles?